src/virtual_elements.js

```diff
@@ -202,6 +202,7 @@ var elementOpen = function(tag, key, statics, var_args) {
   }
 
   firstChild();
+  return node;
```

Comment:
The function should have something like "@return {!Element} The corresponding Element."